examples/Rudaina_work_area/genzsprtwo_21_travel_time.py

```diff
@@ -0,0 +1,48 @@
+import requests
+
+"""
+
+origin: user input of origin place
+destination: user input of distination
+return: estimated travel time between the 2 destnatiosn
```

Comment:
In Python, We usually put the doc string after defining the function signature.  PyCharm will recognize triple double quotes, once they are put after the signature of the function